networkmanager/networkmanager.go

```diff
@@ -208,6 +217,28 @@ func New(cfg *config.Config, trafficStorage TrafficStorage) (manager *NetworkMan
 		if err != nil {
 			return manager, err
 		}
+
+		manager.pollTimer = time.NewTicker(UpdateIptablesCachePeriod)
```

Comment:
It would be more logical to move handling polling timer inside traffic instance?

Reply:
Done